sdk/communication/communication-call-automation/CHANGELOG.md

```diff
@@ -1,18 +1,27 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.3.0-beta.1 (Unreleased)
 
 ### Features Added
-- EventProcessor is a new way to easily handle mid-connection events. Take a look at readme file for usage.
-- StartRecording now accepts PauseOnStart.
-- Introduced `TeamsComplianceRecordingStateChanged` and `TeamsRecordingStateChanged` events.
 
 ### Breaking Changes
 
 ### Bugs Fixed
 
 ### Other Changes
 
+## 1.2.0 (2024-04-15)
+
+### Features Added
+
+- Support for Bring Your Own Storage recording option
+- Support for PauseOnStart recording option 
+- Support for Recording state change with new recording kind's
```

Comment:
can we add links to the PRs that added these changes?

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/29710

Reply:
I mean adding the link in the changelog. You can take a look at tother libraries in the repo for an example of how to link changes to their PRs

Reply:
@joheredi by any chance do you remember which SDK has the link to the PR for the feature? I dont see atleast on the communication sdk's changelog i dont see them included any links.